python/tests/test_logistic_regression.py

```diff
@@ -1937,6 +1937,8 @@ def test_standardization_sparse_example(
     reg_factors: Tuple[float, float],
     float32_inputs: bool = False,
 ) -> None:
+    _convert_index = "int32" if random.choice([True, False]) is True else "int64"
```

Comment:
Can we make this deterministic?

Reply:
Done

Reply:
You shouldn't set the seed each time  just before the choice.  Then it will always be the same choice for each set of params for which the test is run.    I assume you want some variation in this so, need to set only once before testing through all the parameters is kicked off.

Reply:
Nice finding. Revised several places.